__test__/item.test.js

```diff
@@ -0,0 +1,17 @@
+var Shop = require('../src/shop');
```

Comment:
It looks like your README.md for setting up jest is outdated. Can you update your README?

Reply:
Ok.